README.md

```diff
@@ -65,23 +79,39 @@ var server = new ServerMock({
 #### `start(callback)`
 
 Starts the server and invokes the callback once ready to accept connections.
+An error object is passed to the callback if an error occurred. 
 
 Example:
 ```js
 beforeEach(function(done) {
-    server.start(done);
+    server.start(function(err) {
+        if(err) {
+            console.error(err);
+            process.exit(-1);
```

Comment:
I would not promote the usage of `process.exit()` in case a before or after function fails. The `done` callback should support the optional error object to stop the execution, so maybe you can replace the whole callback function with:

```
server.start(done);
```

May you check if it works as expected, please?

_Same exact comment for the `stop()`._